starlark/src/eval/expr.rs

```diff
@@ -206,6 +218,47 @@ impl ExprCompiled {
     }
 }
 
+impl Inspectable for ExprCompiled {
+    fn inspect(&self) -> Value {
```

Comment:
Can you add some unit test for this? Constructing some Inspectable and calling the inspect on it...